src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.c

```diff
@@ -135,23 +135,23 @@ void func_809F6730(EnDodojr* this, PlayState* play, Vec3f* arg2) {
     func_8002836C(play, &pos, &velocity, &accel, &prim, &env, 100, 60, 8);
 }
 
-s32 func_809F68B0(EnDodojr* this, PlayState* play) {
+s32 EnDodojr_UpdateRemainingBounces(EnDodojr* this, PlayState* play) {
     if (this->actor.velocity.y >= 0.0f) {
         return 0;
     }
 
-    if (this->unk_1FC == 0) {
+    if (this->bounceCounter == 0) {
         return 0;
     }
 
     if (this->actor.bgCheckFlags & BGCHECKFLAG_GROUND) {
         Audio_PlayActorSound2(&this->actor, NA_SE_EN_DODO_M_GND);
         this->dustPos = this->actor.world.pos;
-        func_809F6510(this, play, 10);
-        this->actor.velocity.y = 10.0f / (4 - this->unk_1FC);
-        this->unk_1FC--;
+        EnDodojr_SpawnLargeDust(this, play, 10);
+        this->actor.velocity.y = 10.0f / (4 - this->bounceCounter);
+        this->bounceCounter--;
 
-        if (this->unk_1FC == 0) {
+        if (this->bounceCounter == 0) {
             this->actor.velocity.y = 0.0f;
             return 1;
```

Comment:
this function seems to return a boolean
```suggestion
            return true;
```

Reply:
[9fab0f5](https://github.com/zeldaret/oot/pull/1311/commits/9fab0f53142d72db7194c4340ecbe40a0d8fed6c)